Lib/test/test_unicode.py

```diff
@@ -17,6 +17,37 @@
 from test import support, string_tests
 from test.support.script_helper import assert_python_failure
 
+# First invalid code point past the valid range
+INVALID_CODEPOINT = sys.maxunicode + 1
+
+# Characters in the ASCII range (U+0000-U+007f)
+ASCII_CHAR = "a"
+ASCII_LAST_CHAR = "\x7f"
+# Characters in the UCS1 ("latin1") range (U+0000-U+00ff),
+# but not in the ASCII range
+UCS1_CHAR = "\xe9"
+UCS1_LAST_CHAR = "\xff"
+# Characters in UCS2 ("BMP") range (U+0000-U+FFFF),
+# but not in the UCS1 range
```

Comment:
In the normative Unicode jargon, UCS-2 isn't a set of characters -- it's an encoding. This range of characters (*) is called the BMP ("Basic Multilingual Plane"). UCS-2 is a partial encoding, which only works for the BMP.

See https://unicode.org/glossary/

This is all really about characters (or code points), not encodings. So I think the talk of UCS-2 is confusing here -- it's best to just say "BMP", which is the proper name of this range of characters.

(*) in fact of code points, but that doesn't matter here